src/drivers/QtTestDriver.cpp

```diff
@@ -0,0 +1,25 @@
+#include "cucumber-cpp/internal/drivers/QtTestDriver.hpp"
+#include "cucumber-cpp/internal/utils/qtCapture.hpp"
+
+#include <QtTest/QtTest>
+
+namespace cucumber {
+namespace internal {
+
+const InvokeResult QtTestStep::invokeStepBody() {
+    QtTestObject testObject(this);
+
+    qtCapture::Init();
+    qtCapture::BeginCapture();
```

Comment:
unfortunately only to file or stdout and we need capturing output to get results